src/script/interpreter.cpp

```diff
@@ -2282,6 +2295,9 @@ static bool ExecuteWitnessScript(const Span<const valtype>& stack_span, const CS
                 // Note how this condition would not be reached if an unknown OP_SUCCESSx was found
                 return set_error(serror, SCRIPT_ERR_BAD_OPCODE);
             }
+            if ((flags & SCRIPT_VERIFY_LNHANCE) && opcode == OP_INTERNALKEY) {
```

Comment:
this should probably be part of the if else below

Reply:
ah, yeah, slightly different variants of the same pattern. No difference in behavior though. Will change to match mutiny style. :-D